build.gradle

```diff
@@ -1,14 +1,16 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
+    ext.kotlin_version = '1.2.51'
```

Comment:
I think it's not relate to current issue

Reply:
I check and update PR in several hours.